atom-manager-play-lib/build.sbt

```diff
@@ -5,7 +5,7 @@ name := "atom-manager-play"
 libraryDependencies ++= Seq(
   "com.typesafe.play"      %% "play"                  % playVersion,
   "com.gu"                 %% "content-atom-model"    % contentAtomVersion,
-  "org.scalatestplus.play" %% "scalatestplus-play"    % "5.1.0" % Test,
+  "org.scalatestplus.play" %% "scalatestplus-play"    % "6.0.1" % Test,
```

Comment:
This is the [recommended version](https://github.com/playframework/scalatestplus-play?tab=readme-ov-file#releases) of `scalatestplus-play` for Play 2.9:

![image](https://github.com/guardian/atom-maker/assets/52038/e1e96f06-c564-4e3b-9195-e806dc1f28f0)